utility/consistency/theta_h0.py

```diff
@@ -42,20 +44,23 @@ def H0_to_theta(hubble, omega_nu, omnuh2, omega_m, ommh2, omega_c, omch2, omega_
                         num_massive_neutrinos=num_massive_neutrinos,
                         nnu=nnu,
                         H0=hubble)
+        p.set_dark_energy(w=w, wa=wa, dark_energy_model='ppf')
         r = camb.get_background(p)
         theta = r.cosmomc_theta()
     except:
         theta = np.nan
     finally:
         camb.config.FeedbackLevel = original_feedback_level
-    
     return theta
 
 
 def H0_to_theta_interface(params):
     hubble = params['hubble']
     omega_nu = params.get('omega_nu', np.nan)
     omnuh2 = params.get('omnuh2', np.nan)
+    mnu = params.get('mnu', np.nan)
+    TCMB = params.get('TCMB', np.nan)
+    omnuh2 = params.get('omnuh2', np.nan)
```

Comment:
this is a repeated line?
`omnuh2 = params.get('omnuh2', np.nan)`

Reply:
Thanks! Fixed.